source/transfer_list.rst

```diff
@@ -130,15 +130,15 @@ field.
 
    * - 31:1
      - unused
-     - Reserved for future use. Must be 0 or ignored.
+     - Reserved for future use. Must be 0.
```

Comment:
Not sure about this change. I think the intent is to say the writers must set to 0 and readers must ignore this. So if anything this should be more descriptive. But perhaps that's out of scope for this PR.

Reply:
Reverted this change.